examples/csv_to_schemaorg.py

```diff
@@ -1,37 +1,58 @@
+#!/usr/bin/env python3
+
 import os
+import re
 import json
+import argparse
 
 import pandas as pd
 
 from schematic.schemas.explorer import SchemaExplorer
-
 from schematic.utils.csv_utils import create_schema_classes
-from schematic.utils.config_utils import load_yaml
-
-from definitions import CONFIG_PATH, DATA_PATH
-
-config_data = load_yaml(CONFIG_PATH)
-
-# path to base schema
-base_schema_path = os.path.join(DATA_PATH, '', config_data["model"]["biothings"]["location"])
-
-# schema name (used to name schema json-ld file as well)
-output_schema_name = "HTAN"
-
-# schema extension definition csv files
-schema_extensions_csv = [
-                        os.path.join(DATA_PATH, '', 'csv/HTAN.csv')
-                        ]
+from schematic import CONFIG
+
+# Constants (to avoid magic numbers)
+FIRST = 0
+
+# Try to find BioThings JSON-LD file relative to script
+script_dir = os.path.dirname(os.path.realpath(__file__))
+biothings  = os.path.join(script_dir, "..", "data", "schema_org_schemas",
+                          "biothings.jsonld")
+is_biothings_absent = not os.path.exists(biothings)
+
+# Create command-line argument parser
+parser = argparse.ArgumentParser()
+parser.add_argument("schema_csv_list", nargs="+", metavar="schema_csv",
+                    help="Input CSV schema files.")
+parser.add_argument("--output_jsonld", "-o", help="Output JSON-LD schema file.")
+parser.add_argument("--config", "-c", help="Configuration YAML file.")
+args = parser.parse_args()
+
+# Load configuration
+config_data = CONFIG.load_config(args.config)
+biothings_jsonld = CONFIG["model"]["biothings"]["location"]
+base_schema_path = os.path.join(CONFIG.DATA_PATH, biothings_jsonld)
```

Comment:
This is what I imagine would be in every user-facing `schematic` script. Alternatively, the user can specify the config using the `SCHEMATIC_CONFIG` environment variable (see comments in `schematic/configuration.py` for more details).